src/components/VideoCard/VideoCard.jsx

```diff
@@ -0,0 +1,62 @@
+import propTypes from 'prop-types';
+import React from 'react';
+import { getIcons, getLastPosted, getStringValue } from '../../util';
+import './VideoCard.css';
+function VideoCard({
+  channelThumbnail,
+  creator,
+  description,
+  imageThumbnail,
+  likes,
+  releaseDate,
+  subscribers,
+  title,
+  views,
+  youtubeId,
+  category,
+  _id,
+}) {
+  return (
+    <>
+      <div className="video-card">
+        <div className="video-thumbnail-wrapper">
+          <img
+            className="responsive-img video-thumbnail"
+            src={imageThumbnail}
+          />
+        </div>
+
+        <div className="video-card-footer">
+          <img className="video-card-channel" src={channelThumbnail} />
+          <div className="card-text">
+            <div className="card-title">{title}</div>
+            <div className="text-muted">{creator}</div>
```

Comment:
Is there a reason for enclosing the title and creator in `div` rather than an `h5 or h6` or `p` tag? You could avoid these and try to work with semantic HTML as much as possible.